cmd/pullrequest.go

```diff
@@ -43,4 +47,24 @@ var pullrequestCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(pullrequestCmd)
+	
+	pullrequestCmd.Flags().StringVarP(
+		&LinkType,
+		"link-type",
+		"l",
+		"close",
+		"The type of link to create with the associated issue.",
+	)
+}
+
+func validateLinkType() {
+	validLinkTypes := map[string]bool {
+		"close": true,
+		"connect": true,
+	}
+
+	if !validLinkTypes[LinkType] {
+		err := fmt.Sprintf("Invalid link type: %s. Must be close or connect.")
+		log.Fatalln(err)
```

Comment:
There's also `log.Fatalf`.

PS: You're not using the `%s` in the formatting afaict as well.

Reply:
Oops! This was also addressed in the move to a util class.